cmd/gosh-query/main.go

```diff
@@ -90,7 +90,7 @@ func main() {
 		log.WithError(err).WithField("path", storep).Fatal("Failed to load store")
 	}
 
-	store, err := internal.NewStore(storep, false)
+	store, err := internal.NewStore(storep, false, false)
```

Comment:
Well, sine the whole point is that the operator shouldn't have access to the secret key, I would say that this is actually intended behaviour. You should still be able to query the metadata, just with the encrypted filename.

What would you say that the `gosh-query` command should do when encountering an encrypted store?

Reply:
I'm not sure, but I think with the currently proposed changes `gosh-query` won't be able to delete items from an encrypted store. However, as I stated before, this can be ignored as this tool should be integrated with `goshd` anyway.

Reply:
okay, so this is resolved then, I guess?

Reply:
:shrug: